README.md

```diff
@@ -191,19 +191,44 @@ CLOUDFLARE_APPLICATION_AUDIENCE=<CLOUFLARE_ACCESS_AUDIENCE_TAG>
 
 ### Docker Build and Run
 
-Build the Docker image
+Build the Docker image and run it using Docker Compose:
 
 ```
-docker build -t access .
+docker-compose up --build
 ```
 
-Run it using your `.env-production` variables
+The command above will build and run the container
 
-```
-docker run --rm --env-file .env-production -p 3000:3000 access
-```
+Go to [http://localhost:3000/](http://localhost:3000/) to view the application
+
+### Docker configuration
+
+Before launching the container with Docker, make sure to configure `.env.psql` and `.env.production`:
+
+#### Configuration for `.env.psql`
+
+The `.env.psql` file is where you configure the PostgreSQL server credentials, which is also Dockerized.
+
+- `POSTGRES_USER`: Specifies the username for the PostgreSQL server.
+- `POSTGRES_PASSWORD`: Specifies the password for the PostgreSQL server.
+
+#### Configuration for `.env.production`
+
+The `.env.production` file is where you configure the application.
+
+- `OKTA_DOMAIN`: Specifies the [Okta](https://okta.com) domain to use.
+- `OKTA_API_TOKEN`: Specifies the [Okta](https://okta.com) [API Token](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApiToken/) to use.
+- `DATABASE_URI`: Specifies the Database connection URI. **Example:** `postgresql://<host>:<user>@<password>:5432/<db_name>`
+- `CLIENT_ORIGIN_URL`: Specifies the origin URL which is used by CORS.
+- `REACT_APP_API_SERVER_URL`: Specifies the API base URL which is used by the frontend.
+- `FLASK_SENTRY_DSN`: See the [Sentry documentation](https://docs.sentry.io/product/sentry-basics/concepts/dsn-explainer/).
+- `REACT_SENTRY_DSN`: See the [Sentry documentation](https://docs.sentry.io/product/sentry-basics/concepts/dsn-explainer/).
```

Comment:
Also `FLASK_SENTRY_DSN` and `REACT_SENTRY_DSN` are entirely optional, but great to set if you use Sentry.io